template/README.md

```diff
@@ -0,0 +1,35 @@
+## Template 
+This project includes all common dependencies and settings for kaluga subproject.
+
+#Steps for adding new subproject
+1. CopyPaste `teplete` subproject, select name for your subproject (we recommend to use only lowercase characters).
+2. `<your subproject/>src/androidLibMain/AndroidManifest.xml` change `com.splendo.kaluga.template` to your subproject package name.	
+3. `<your subproject/>src/androidLibAndroidTests/AndroidManifest.xml` change `com.splendo.kaluga.template` to your subproject package name.
+4. `<your subproject/>src/androidLibAndroidTests/kotlin/TestActivity.kt` change package name from `com.splendo.kaluga.template` to `com.splendo.kaluga.your module`.
+5. Include your subproject to kaluga project
+	go to `kaluga/settings.gradle/kts`
+	And add in the end of the file:
+	`include(":<your subproject>")`.
+6. Add your source code. For more derails check README.md in source root and each platform folder.
+7. Remove this markdown file from your subproject.
+8. Don't forget to add copyright if files were imported from another project
```

Comment:
FYI, this copyright template already embedded into project and you can easily update 'licence' for selected files